packages/terra-application-links/examples/index-examples/ApplicationLinkConfig.js

```diff
@@ -0,0 +1,16 @@
+const applicationLinkConfig = [
+  { path: '/durp', text: 'durp' },
```

Comment:
Let's come up with better sample content.

Reply:
Ha Touche! We don't have a legitimate use-case yet, so I just made up random words. Any recommendations would be great.

Reply:
Maybe sync with Tim or Paul?